packages/bpk-component-calendar/src/BpkColoredCalendarDate.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import BpkCalendarDate, { propTypes, defaultProps } from './BpkCalendarDate';
+import STYLES from './BpkColoredCalendarDate.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const CellType = {
+  default: 'default',
+  negative: 'negative',
+  neutral: 'neutral',
+  positive: 'positive',
+};
+
+type Props = {
+  cellType: String,
```

Comment:
This should be `cellType: $Keys<typeof CellType>`

Reply:
Good spot